sktime/markov/msm/koopman_reweighted_msm.py

```diff
@@ -2,60 +2,72 @@
 
 import numpy as _np
 from msmtools.estimation import effective_count_matrix
-
-from sktime.markov.msm import MarkovStateModel
+from scipy.sparse import issparse
 from sktime.markov._base import _MSMBaseEstimator
+from sktime.markov.msm import MarkovStateModel
 from sktime.markov.transition_counting import TransitionCountEstimator, TransitionCountModel
 from sktime.util import submatrix
-from ._koopman_reweighted_msm_impl import (bootstrapping_count_matrix, bootstrapping_dtrajs, twostep_count_matrix,
-                                           rank_decision, oom_components, equilibrium_transition_matrix)
 
-__author__ = 'Feliks Nüske, Fabian Paul, marscher'
+from . import _koopman_reweighted_msm_impl as _impl
```

Comment:
is there something more readable than `_impl` here?

Reply:
this is an internal implementation import which should not be used publicly so I think this is ok, impl is fairly standard?

Reply:
sounds good to me